examples/voting.ms

```diff
@@ -0,0 +1,18 @@
+pwoli_sadhanam age;
+dhe_pidicho "Input age: ";
+age=number_thada;
+seriyano age um 18 um same_aane
+{
+    dhe_pidicho "You are not eligible to vote.\n";
+}
+seri_allel
+{
+    seriyano 18 nekal age veluthane
```

Comment:
the keyword should be `ne_kal` instead of nekal (may be I should add nekal too in the list of keywords, as part of normlaisation #5

Reply:
ok